easybuild/easyconfigs/n/NCO/NCO-4.7.5-CrayGNU-18.07.eb

```diff
@@ -0,0 +1,33 @@
+# contributed by Luca Marsella (CSCS), updated by Samuel Omlin (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'NCO'
+version = "4.7.5"
+
+homepage = 'http://nco.sourceforge.net/'
+description = """The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://nco.sourceforge.net/src/']
+
+dependencies = [
+    # not using cray-netcdf/4.4.1 because it is compiled with cray-hdf5/1.10
+    # NCO: openmp support built by default, mpi is not yet supported
+    ('ANTLR', '2.7.7', '-python2'),
+   #('netCDF', '4.4.1.1'),
```

Comment:
Please remove these lines with past dependencies instead of just commenting them out, as they are not needed anymore